templates/lookup_identity.conf.erb

```diff
@@ -3,6 +3,7 @@
   LookupUserAttr email REMOTE_USER_EMAIL
   LookupUserAttr firstname REMOTE_USER_FIRSTNAME
   LookupUserAttr lastname REMOTE_USER_LASTNAME
+  LookupUserGroups REMOTE_USER_GROUPS :
   LookupUserGroupsIter REMOTE_USER_GROUP
```

Comment:
Just to verify: the old `REMOTE_USER_GROUP_N` env var is still present, right? That's good to have for backwards compatibility. Just to be clear: I like the new implementation more but I want to understand the impact.

Reply:
Yes I have left the old env variables as is.